src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -374,6 +381,51 @@ class IssueListOverview extends React.Component<Props, State> {
     }
   };
 
+  fetchCounts = async (currentQueryCount: number) => {
+    const {queryCounts: _queryCounts} = this.state;
+    let queryCounts: QueryCounts = { ..._queryCounts }
+
+    const endpointParams: EndpointParams = this.getEndpointParams();
+    const currentTabQuery = endpointParams.query && TabQueriesWithCounts.includes(endpointParams.query as Query)
```

Comment:
```suggestion
    const currentTabQuery = TabQueriesWithCounts.includes(endpointParams.query as Query)
```
not sure if you need to check if `query` exists on endpointParams, even if `endpointParams.query` is undefined the `.includes` will just return false

Reply:
Typescript was giving me an error for `.includes` but `as Query` solves that